src/main/java/sirius/biz/process/output/TableOutput.java

```diff
@@ -136,6 +137,15 @@ public RowBuilder withCell(Cell cell) {
             return this;
         }
 
+        /**
+         * Adds a plain value to a row.
+         * @param data the plain value to add.
+         * @return the row builder itself for fluent method calls
+         */
```

Comment:
The following would probably be closer to the standards used elsewhere in this file.

```suggestion
        /**
         * Adds a plain value to a row.
         *
         * @param data the plain value to add
         * @return the row builder itself for fluent method calls
         */
```